.cirrus.yml

```diff
@@ -221,7 +221,15 @@ task:
         # Native integration tests are handled by firebase-test-lab below, so
         # only run unit tests.
         # Must come after build-examples.
-        - ./script/tool_runner.sh native-test --android --no-integration --exclude script/configs/exclude_native_unit_android.yaml
+        # TODO(stuartmorgan): Remove this condition once
+        # https://github.com/flutter/flutter/issues/96661 is fixed and
+        # cherry picked. Currently webview_flutter tests crash on stable on
+        # Android so must be skipped.
+        - if [[ "$CHANNEL" == "stable" ]]; then
+        -   ./script/tool_runner.sh native-test --android --no-integration --exclude script/configs/exclude_native_unit_android.yaml,webview_flutter
```

Comment:
Does this `if` belong in line 241 of the file? `./script/tool_runner.sh firebase-test-lab` instead?

Reply:
Doh!